engine/framework/master.go

```diff
@@ -146,6 +164,14 @@ type BaseMaster interface {
 		cost model.RescUnit,
 		resources ...resModel.ResourceID,
 	) (frameModel.WorkerID, error)
+
+	// CreateWorkerV2 is the latest version of CreateWorker, but with
```

Comment:
Will the old `CreateWorker` be removed later?

Reply:
Yes.